src/pages/errors/unauthorised-error.tsx

```diff
@@ -0,0 +1,24 @@
+import { useNavigate } from 'react-router-dom'
+import { Button } from '@/components/custom/button'
+
+export default function UnauthorisedError() {
+  const navigate = useNavigate()
+  return (
+    <div className='h-svh'>
+      <div className='m-auto flex h-full w-full flex-col items-center justify-center gap-2'>
+        <h1 className='text-[7rem] font-bold leading-tight'>401</h1>
+        <span className='font-medium'>Oops! You don't have permission to access this page.</span>
```

Comment:
Can you run `pnpm format`? 
Then we're good to go.

Reply:
Thanks for review.
`pnpm format` done for this file